peer-monitoring-service/client/src/network.rs

```diff
@@ -89,8 +106,13 @@ pub async fn send_request_to_peer(
         peer_network_id,
     );
 
+    let client = {
```

Comment:
Why are we wrapping the client in a RWLock? 🤔

Reply:
Removed